pkg/controllers/certificates/certificate_secret.go

```diff
@@ -36,7 +36,7 @@ func newCertificateSecret(scheme *runtime.Scheme, deployment *appsv1.Deployment)
 }
 
 func (certSecret *certificateSecret) setSecretFromReader(ctx context.Context, apiReader client.Reader, namespace string) error {
-	query := k8ssecret.NewQuery(ctx, nil, apiReader, log)
+	query := k8ssecret.NewQuery(ctx, nil, apiReader, log.Logger)
```

Comment:
`k8ssecret.NewQuery` is also our code, so we can expect our logger (https://github.com/Dynatrace/dynatrace-operator/blob/9a3bcd1bc6b3eb9f946d056c83474efb23dde74b/pkg/util/kubeobjects/secret/secret.go#L26-L26)